test/typescript/test.ts

```diff
@@ -4,7 +4,7 @@ import { check, checkDirectory } from 'typings-tester';
 describe('typescript typings tests', () => {
   it('should allow WebClient to work without a token', () => {
     check([`${__dirname}/sources/webclient-no-token.ts`], `${__dirname}/tsconfig-strict.json`);
-  });
```

Comment:
maybe the best place to override this value is in `test/mocha.opts`, so that it doesn't just apply to this one case. i don't see anything unique to this case that should make it run longer than the other. its my guess that Travis generally has unpredictable speed and we should make the ceiling high enough to accomodate.

Reply:
Agree! I could not find this setting before.